CMakeLists.txt

```diff
@@ -131,10 +131,7 @@ endif()
 
 set(CURRENT_VERSION "Uncrustify-0.70.0_f")
 
-option(NoGitVersionString "Do not use make_version.py and git to build a version string" OFF)
-if(NoGitVersionString)
-  configure_file(src/uncrustify_version.h.in uncrustify_version.h @ONLY)
-else()
+if(EXISTS "../.git")
```

Comment:
This shouldn't be needed. What issue are you seeing? Non-repo builds seem to work fine for me...

Reply:
Well, it doesn't work at me. So I try this test.
Give me time enought to make a complete new test.

Reply:
Can you share what you *are* seeing?